src/i18n/strings/en_EN.json

```diff
@@ -862,7 +862,7 @@
     "Error: Problem communicating with the given homeserver.": "Error: Problem communicating with the given homeserver.",
     "Can't connect to homeserver via HTTP when an HTTPS URL is in your browser bar. Either use HTTPS or <a>enable unsafe scripts</a>.": "Can't connect to homeserver via HTTP when an HTTPS URL is in your browser bar. Either use HTTPS or <a>enable unsafe scripts</a>.",
     "Can't connect to homeserver - please check your connectivity, ensure your <a>homeserver's SSL certificate</a> is trusted, and that a browser extension is not blocking requests.": "Can't connect to homeserver - please check your connectivity, ensure your <a>homeserver's SSL certificate</a> is trusted, and that a browser extension is not blocking requests.",
-    "Sorry, this homeserver is using a login which is not recognised ": "Sorry, this homeserver is using a login which is not recognised ",
+    "This homeserver doesn't offer any login flows which are supported by this client.": "This homeserver doesn't offer any login flows which are supported by this client.",
```

Comment:
Could this include some information about what a user should do in order to solve the problem? E.g. "Please make sure you are running the latest version of Riot, and check the website of the homeserver to see if they recommend using a specific client."

Reply:
you're assuming that the homeserver has a website?

Reply:
Ideally a homeserver could provide the client with things like links to privacy policy, support page, etc. but barring that, it seems like there is at least no harm in suggesting looking for a page that may or may not exist. A user that connects to a homeserver not ran by an organization of some sort (who usually have websites) probably has the skill level to avoid the advice.

Reply:
IMHO we're really just saying "this client doesn't support this homeserver", but trying to give the expert user a bit more information on why not.

Riot is first and foremost a hosted webapp, so "Please make sure you are running the latest version of Riot" is nonsensical for the average user. Plus this is react-sdk, not Riot, so we can't necessarily assume the user is using Riot.

In general a user will have had to register with a homeserver to get this far, which suggests that they have, in the past, used a client which supports whatever weird login flow the server wants, so they can't be completely clueless.

Reply:
Good point. Except the server might have changed their login flow, which means the client they have been using no longer works.